.github/dependabot.yml

```diff
@@ -28,6 +28,28 @@ updates:
     pull-request-branch-name:
       separator: "-" 
 
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+    labels:
+      - "dependencies"
```

Comment:
Should we be adding the `Needs review` label to all automated actions as well? If someoene filters PRs based on their review status, this could be helpful.

Reply:
guess we could add that to all of them.